yarnrunner_python/runner.py

```diff
@@ -38,9 +38,21 @@ def __init__(self, compiled_yarn_f, names_csv_f, autostart=True, enable_tracing=
         self.paused = True
         self.finished = False
 
+        self.load_initial_vars()
+
         if autostart:
             self.resume()
 
+    def load_initial_vars(self):
+        for key in self._compiled_yarn.initial_values:
+            initial_value = self._compiled_yarn.initial_values[key]
+            if len(initial_value.string_value) > 0:
+                self.variables[key] = initial_value.string_value
+            elif initial_value.float_value != 0:
+                self.variables[key] = initial_value.float_value
+            else:
+                self.variables[key] = initial_value.bool_value
+
```

Comment:
this piece of code here is awful and doesn't supports initializing string variables with empty strings and initializing numeric variables with 0, and will default them to the bool_value (false).

there is a better way to identify the type of an operand?